src/components/product/ProductCard.jsx

```diff
@@ -0,0 +1,79 @@
+/** @jsxImportSource @emotion/react */
+import { css } from '@emotion/react';
+import { priceFunc } from '@/src/utils/utils.js';
+import c from '@/src/utils/constants.js';
+import Image from 'next/image';
+
+const style = {
+  card: css`
+    display: grid;
+    gap: 1.6rem;
+    grid-template-rows: 1fr auto;
+
+    .imageWrapper {
+      width: 22.1rem;
+      aspect-ratio: 1/1;
+      position: relative;
+
+      border-radius: 16px;
+
+      @media (max-width: ${c.BREAKPOINTS.TABLET}px) {
+        &.best {
+          width: 343px;
+          border-radius: 19.46px;
+        }
+      }
+
+      @media (max-width: ${c.BREAKPOINTS.MOBILE}px) {
+        width: 16.8rem;
+        border-radius: 12px;
+      }
+    }
+  `,
+  info: css`
+    height: 8rem;
+
+    display: flex;
+    flex-direction: column;
+    gap: 0.6rem;
+  `,
+  title: css`
+    font-weight: 500;
+    font-size: 1.4rem;
+    line-height: 2.4rem;
+    color: var(--gray-800);
+  `,
+  price: css`
+    font-weight: 700;
+    font-size: 1.6rem;
+    line-height: 2.6rem;
+    color: var(--gray-800);
+  `,
+  like: css`
+    font-weight: 500;
+    font-size: 1.2rem;
+    line-height: 1.8rem;
+    color: var(--gray-600);
+  `,
+};
+
+export default function ProductCard({ item, best = false }) {
+  const { likeCount, price, name, images } = item;
+  const imgUrl = images?.length ? images[0] : '/Image/img_default.png';
```

Comment:
@csbizz 요렇게 줄일수 있겠네요
```suggestion
  const imgUrl = images?.[0] || '/Image/img_default.png';
```

Reply:
배열의 경우에도 optional chaining을 지원하는군요? 처음 알았습니다;

Reply:
@csbizz 넵 아실수도있는데 배열말고도 함수에도 적용할 수 있고, 다양하게 사용할 수 있죵
```js
array?.sort?.()
```